docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -1564,6 +1564,38 @@ Upload a tar archive to be extracted to a path in the filesystem of container
     - no such file or directory (**path** resource does not exist)
 - **500** – server error
 
+
+### Prune stopped containers
+
+`POST /containers/prune`
+
+Delete stopped containers
+
+**Example request**:
+
+    POST /containers/prune HTTP/1.1
+    Content-Type: application/json
+
+    {
```

Comment:
`{}` it stills expect a struct. Atm it's empty but it'll fill up with the options that'll be added later (e.g. filters)